cmd/kubeadm/app/util/config/testdata/validation/invalid_unknown_field_nodecfg.yaml

```diff
@@ -0,0 +1,3 @@
+apiVersion: kubeadm.k8s.io/v1beta1
+kind: NodeConfiguration
```

Comment:
Again, file should be named `invalid_unknown_field_joincfg.yaml` and kind should be `JoinConfiguration`.